sgt-back-end/remove.js

```diff
@@ -0,0 +1,41 @@
+const db = require('./db');
+const { jsonError, sendStatus } = require('./status');
+const errorCheck = require('./errorCheck');
+
+function remove(app) {
+  app.delete('/api/grades/:gradeId', (req, res, next) => {
+
+    const sql = `
+      DELETE
+          FROM "grades"
+        WHERE "gradeId" = $1
+      RETURNING *
+    ;
+  `;
+
+    const gradeId = Number(req.params.gradeId);
+    const params = [gradeId];
+    const course = null;
```

Comment:
I should talk to the class about `null`. In my opinion (and that of many others, but certainly not everyone), `null` should never have been added to the language. I _always_ use `undefined` unless some other code I'm using requires me to use `null`.